src/pages/MySolutions.tsx

```diff
@@ -175,6 +175,28 @@ export const mySolutionsQuery = graphql`
                   }
                 }
               }
+              files {
```

Comment:
Also @0qy& @chrisbc let's see if we can help this by having the API resolve a **ScaledInversionSolution** (for AutomationTask.inversion_solution (rather than files .....)